src/interface/interface.tsx

```diff
@@ -0,0 +1,29 @@
+export interface UserInfo {
+  userId: number;
+  userName: string;
+}
+
+export interface MessageInfo {
+  addText: string;
+  messageId: number;
+  userNum: number;
+}
+// export interface ExtendedMessageInfo extends MessageInfo {
+//   messageId: number;
+// }
+// export interface ListInfo {
+//   chattingRoomId: number;
+//   addText: string;
+//   userNum: number;
+//   message: Message[]; // message 가져오자
+// }
+
+export interface ChatInfo {
+  addText: string;
+  userNum: number;
+}
```

Comment:
요기 두개는 messageId를 제외하고는 같은 구조여서 의도에 따라 다르겠지만, 
```suggestion
export interface MessageInfo {
  addText: string;
  userNum: number;
  messageId?: number;
}
```
와 같이 축약할 수도 있을 것 같아요